.ci_support/linux_64_.yaml

```diff
@@ -32,6 +32,8 @@ glib:
 - '2'
 lcms2:
 - '2'
+libhwy:
+- '1.1'
```

Comment:
this kind of pinning is important to get a consistent version of libhwy accorss conda forge.